internal/app/machined/pkg/controllers/runtime/kernel_param_defaults.go

```diff
@@ -104,11 +104,38 @@ func (ctrl *KernelParamDefaultsController) getKernelParams() []*kernel.Param {
 			Key:   "net.ipv6.conf.default.accept_ra",
 			Value: "2",
 		},
+		// ipvs/conntrack tcp keepalive refresh.
+		{
+			Key:   "net.ipv4.tcp_keepalive_time",
+			Value: "600",
+		},
+		{
+			Key:   "net.ipv4.tcp_keepalive_intvl",
+			Value: "60",
+		},
+		// RFC1323 disable.
+		{
+			Key:   "net.ipv4.tcp_timestamps",
```

Comment:
I'm not sure if this is good, on recent Linux timestamps are randomized, so it shouldn't be a security issue anymore 

e.g. https://security.stackexchange.com/questions/111794/pros-and-cons-of-disabling-tcp-timestamps